### PR TITLE
T7 45 parse ticketmaster category tc

### PR DESCRIPTION
**Pull request by mchen132**
Merge t7-45parse-ticketmaster-category-TC branch to feature/iteration-3

### CODE REVIEW

File: service/src/main/java/com/TWCC/api/ResponseParser.java [hunk 1]
```diff
@@ -375,6 +380,81 @@ private int extractAgeLimit(Map<String, Object> eventMap) {
...
+        @SuppressWarnings("unchecked")
+        String subGenre = (
+                            (TextNode)
+                                (
+                                    (HashMap<String, Object>)
+                                        categoryMap.get("subGenre")
+                                ).get("name")
+                        ).toString();
+
+        category += "," + subGenre;
```
Comment (JourneyToTheWes) [inline service/src/main/java/com/TWCC/api/ResponseParser.java:453]: Is there only one `genre` and `subGenre` that we're adding to category? If I remember correctly, you showed me multiple genres and subgenres, but the code here only grabs one value?

**Review by JourneyToTheWes — CHANGES_REQUESTED** [review]

File: service/src/main/java/com/TWCC/api/ResponseParser.java (hunk 1 above)
Comment (mchen132) [reply service/src/main/java/com/TWCC/api/ResponseParser.java:453]: The response's combines all the the category-related info into an attribute named `classifications`, and all the sample data I have seen only has one entry in the `classification` attribute, which contains one `segment`, one `genre`, and one `subGenre`. `classification` entries also contains `type` and `subType` attributes, but I deemed these attributes irrelevant to the categories we are looking for. 

Here is a sample `classification` attribute:
```
"classifications": [
                    {
                        "primary": true,
                        "segment": {
                            "id": "KZFzniwnSyZfZ7v7nE",
                            "name": "Sports"
                        },
                        "genre": {
                            "id": "KnvZfZ7vAde",
                            "name": "Basketball"
                        },
                        "subGenre": {
                            "id": "KZazBEonSMnZfZ7vFJA",
                            "name": "NBA"
                        },
                        "type": {
                            "id": "KZAyXgnZfZ7v7l1",
                            "name": "Group"
                        },
                        "subType": {
                            "id": "KZFzBErXgnZfZ7vA7d",
                            "name": "Team"
                        },
                        "family": false
                    }
                ]
```

Comment (JourneyToTheWes) [reply service/src/main/java/com/TWCC/api/ResponseParser.java:453]: I see. Weren't we going with type and subtype too?

Comment (JourneyToTheWes) [reply service/src/main/java/com/TWCC/api/ResponseParser.java:453]: Resolved after discussing not as useful values to be included

**Review by JourneyToTheWes — APPROVED** [review]

**Pull request merged**